packages/remix-utils/src/Link.tsx

```diff
@@ -1,4 +1,4 @@
-import ClickableStyle from '@chanzuckerberg/eds/lib/components/ClickableStyle';
+import {ClickableStyle} from '@chanzuckerberg/eds';
 import type {ClickableStyleProps} from '@chanzuckerberg/eds/lib/components/ClickableStyle';
```

Comment:
does this type import still work?

Reply:
Yup, the type imports still work

Reply:
Yup type imports still work since types are not modules.
Following screenshot is from [edu-stack with alpha publish of the build changes](https://github.com/chanzuckerberg/edu-stack/pull/68)
![image](https://user-images.githubusercontent.com/86632227/229917281-20d4694c-8695-426e-9eb1-613c22186b49.png)